packages/itwinui-react/src/core/Tile/TileNew.tsx

```diff
@@ -0,0 +1,646 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import React from 'react';
+import cx from 'classnames';
+import {
+  useSafeContext,
+  // useTheme,
```

Comment:
This can be removed. Also, you can import `useGlobals` and call that within `TileComponent`

Reply:
Right yea. I forgot to clean up comment. Updated

Reply:
Make sure to call `useGlobals` in the `TileComponent`

Reply:
Oop right. Updated :D